layouts/default.vue

```diff
@@ -9,15 +9,25 @@
 <script>
 import sparcHeader from '@/components/header/Header.vue'
 import sparcFooter from '@/components/footer/Footer.vue'
+import { mapState } from 'vuex'
 
 export default {
   components: {
     sparcHeader,
     sparcFooter
+  },
+  computed: {
+    ...mapState({
+      scrollingState: 'disableScrolling'
+    })
   }
 }
 </script>
 
 <style lang="scss" scoped>
 @import '@/assets/_variables.scss';
+.layout {
```

Comment:
This needs to be in a media query to prevent it from messed up on large viewports, as the original problem is still happening.

Steps to reproduce

1. Make viewport small enough to show the mobile header
2. Open mobile menu
3. Make viewport large enough to show desktop header

Reply:
So I was aware of this happening, but I was thinking about it and isn't it not possible for a user to switch to larger viewpoirts on a mobile device? i.e. if a user were on a mobile device and then were to switch to a desktop computer to view the site this issue would no longer be happening. I can add the media query but just a thought.

Reply:
This is true, but we should not code for devices, we should code for viewport sizes. This way, we use the mindset of "Does it look good at this size?" rather than "Does it look good on this device?" This also allows us to subscribe to the idea of responsive design for when your design breaks, and not just for N number of sizes. We don't have to worry about the latter on this site as much, but I've have about 13 media queries for one page for some of my other sites in the past because we coded to when the design broke at a certain viewport size.

We cannot control what size the user has their browser on desktop, and having the user look at it on multiple sizes (while resizing) or at smaller sizes on desktop is a possibility.